README.md

```diff
@@ -1,43 +1,87 @@
-# bot2
+# Bot2
 
+A discord bot made with [interactions.py](https://github.com/interactions-py/interactions.py).
+Visit [the official website](https://interactions-py.github.io/interactions.py/) to get started.
 
+## Development installation
 
+<details>
+<summary>Click to expand</summary>
 
-A discord bot made with [interactions.py](https://github.com/interactions-py/interactions.py).
-Visit [the official website](https://interactions-py.github.io/interactions.py/) to get started.
+### Mongodb
+
+You can either use a local mongodb instance or use mongodb atlas.
 
-# Running the Application
-There are multiple ways to launch the application.
+#### Local
 
+1. Install mongodb on your machine ([Windows](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/),
+   [Mac](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/),
+   [Linux](https://docs.mongodb.com/manual/administration/install-on-linux/))
+2. Create a database called `bot2`
 
 ### Python
-To start the bot with python, you first need to install the required packages with either `poetry install` or `pip install -r requirements.txt` 
 
+0. Clone this repository and, if needed, check out the appropriate feature branch.
+
+0.1. You will need your own Discord bot account with a token. See [here](https://interactions-py.github.io/interactions.py/Guides/02%20Creating%20Your%20Bot/).
```

Comment:
i think you should also link discord's guide as well https://discord.com/developers/docs/getting-started (you can keep discord.py's) since discord's will get updated first, and there's a possibility that this one will get updated late/never.